spk/itools/Makefile

```diff
@@ -23,4 +23,18 @@ SERVICE_SETUP   = src/service-setup.sh
 SERVICE_COMMAND = $${SYNOPKG_PKGDEST}/sbin/usbmuxd
 STARTABLE       = yes
 
+POST_STRIP_TARGET = ios_access_extra_install
+
 include ../../mk/spksrc.spk.mk
+
+.PHONY: ios_access_extra_install
+ios_access_extra_install:
```

Comment:
In case of inspection of make file logs with failures, it is difficult to known "ios_access == itools". So please rename as "itools_extra_instrall"